packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -495,37 +490,47 @@ export default class BitbucketBackend implements Implementation {
         branches.map(branch => contentKeyFromBranch(branch)),
       );
 
-    const readUnpublishedBranchFile = (contentKey: string) =>
-      this.api!.readUnpublishedBranchFile(contentKey);
+    const ids = await unpublishedEntries(listEntriesKeys);
```

Comment:
Instead of reading all the data of all entries, just return a list of id which `backend,js` can use to read entries